packages/plugin-core/src/components/pods/PodControls.ts

```diff
@@ -302,6 +320,65 @@ export class PodUIControls {
     return id;
   }
 
+  /**
+   * Prompts a lookup that will determine the scope of export.
+   * @param fromSelection set this flag to true if we are using {@link PodExportScope.LinksInSelection}
+   * @param key key of the command. this will be used for lookup provider subscription.
+   * @param logger logger object used by the command.
+   * @returns
+   */
+  public static async promptForScopeLookup(opts: {
+    fromSelection?: boolean;
+    key: string;
+    logger: DLogger;
+  }): Promise<NoteLookupProviderSuccessResp | undefined> {
+    const { fromSelection, key, logger } = opts;
+    const extraButtons = [
+      CopyNoteLinkBtn.create(false),
```

Comment:
can we hide this for pods? lets minimize feature surface area

Reply:
sure. thought these may be useful as a side effect for export pods, but not essential.